Assets/MRTK/Providers/OpenVR/HPMotionController.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.OpenVR.Input
+{
+    /// <summary>
+    /// Open VR Implementation of the Windows Mixed Reality Motion Controllers.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.HPMotionController,
+        new[] { Handedness.Left, Handedness.Right },
+        "Textures/MotionController")]
+    public class HPMotionController : GenericOpenVRController
```

Comment:
Looks like we need some changes to OpenVRDeviceManager as well, to reference this class and `SupportedControllerType.HPMotionController`. It's currently all keyed off the joystick name, so hopefully this new controller has a distinct name!

Reply:
![image](https://user-images.githubusercontent.com/3580640/99134472-bd0c6c80-25d2-11eb-8c55-8887857a6a4f.png)

urg, we might end up needing to do a sub-check for the pid part of the name (0x066A for HP)